src/connector.js

```diff
@@ -599,6 +603,13 @@ function initEngine() {
 			}
 		}
 
+		if ( urlParams.year ) {
+			const year = Number.parseInt( urlParams.year );
+			if ( Number.isInteger( year )  && ( year > 2000 )  && ( year < 2250 )) {          
```

Comment:
@bblaisATcoveo I am curious as to what is the use case that lead to adding this part? `&& ( year < 2250 )`

Reply:
i suggested to have some limit on the number , can be any year above expected lifecycle of the system

Reply:
@igkislev Which is fine, but ideally we don't want to have arbitrary numbers. We need to understand the use cases for which we are putting those numbers, and in my opinion there is no reason for putting the upper bracket (2250) unless there is something wrong with the date generation. CC: @bblaisATcoveo

Reply:
@GormFrank @igkislev Code adjusted below